index.js

```diff
@@ -26,7 +32,7 @@ const formatNum = (number, opts) => {
     else number = 0
   }
 
-  const nf = new Intl.NumberFormat([opts.locale], Object.assign({}, opts, { style: 'decimal' }))
+  const nf = getNumberFormatter(Object.assign({}, opts, { style: 'decimal' }))
```

Comment:
i'm guessing you'll get this during security audit, might save u a min
```suggestion
  const nf = getNumberFormatter(Object.assign(Object.create(null), opts, { style: 'decimal' }))
```